src/app/service/feature-toggles.service.ts

```diff
@@ -0,0 +1,139 @@
+import { Location } from '@angular/common';
+import {ErrorHandler, Inject, Injectable, InjectionToken} from '@angular/core';
+import { Headers, Http } from '@angular/http';
+import {
+  isEqual as deepEqual,
+  sortBy
+} from 'lodash';
+import { AuthenticationService } from 'ngx-login-client';
+import { Observable } from 'rxjs';
+import { Feature } from '../models/feature';
+
+export let FABRIC8_FEATURE_TOGGLES_API_URL = new InjectionToken('fabric8.feature.toggles.api.url');
+
+let featureTogglesApiUrlFactory = () => {
+  return process.env.FABRIC8_FEATURE_TOGGLES_API_URL;
+};
+
+export let featureTogglesApiUrlProvider = {
+  provide: FABRIC8_FEATURE_TOGGLES_API_URL,
+  useFactory: featureTogglesApiUrlFactory
+};
+
+@Injectable()
+export class FeatureTogglesService {
+  private headers = new Headers({ 'Content-Type': 'application/json' });
+  private featureTogglesUrl: string;
```

Comment:
Since this is only assigned in the constructor, you can just mark it read only. If not, you should create get/set for it.

By convention, TypeScript lang starts private variables with an underscore. While this is not required, we have tried to do this.

Reply:
https://angular.io/guide/styleguide#properties-and-methods 
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names
I'm not too in favor of adding `_` for private either. 
I've integrated our other comments tho :)

Reply:
ok, works for me